app/load_fhir_data.js

```diff
@@ -0,0 +1,726 @@
+import Canadarm from 'canadarm';
+import $ from '../lib/jquery.min';
+
+window.ASCVDRisk = window.ASCVDRisk || {};
+
+((() => {
+  const ASCVDRisk = {};
+  const PatientInfo = {};
+  ASCVDRisk.ret = $.Deferred();
+
+  /**
+   * Creates a default Patient model for the application with all undefined fields and
+   * resolves the promise used when retrieving patient information. Also sets to hide the
+   * demo banner for the application.
+   */
+  const setDefaultPatient = () => {
+    PatientInfo.firstName = undefined;
+    PatientInfo.lastName = undefined;
+    PatientInfo.gender = undefined;
+    PatientInfo.dateOfBirth = undefined;
+    PatientInfo.age = undefined;
+
+    const relatedFactors = {};
+    relatedFactors.smoker = undefined;
+    relatedFactors.race = undefined;
+    relatedFactors.hypertensive = undefined;
+    relatedFactors.diabetic = undefined;
+    PatientInfo.relatedFactors = relatedFactors;
+
+    PatientInfo.totalCholesterol = undefined;
+    PatientInfo.hdl = undefined;
+    PatientInfo.systolicBloodPressure = undefined;
+    ASCVDRisk.hideDemoBanner = true;
+    ASCVDRisk.ret.resolve(PatientInfo);
+  };
+  ASCVDRisk.setDefaultPatient = setDefaultPatient;
+
+  /**
+   * Initiate Canadarm logging for the application to output on the console and log specified
```

Comment:
Need to update the comment to may be ->  "Initializer function for Canadarm Logger. 
Initiates the Canadarm logging tool to output logs to the console and the URL specified in the config.js file (in root directory).